index.html

```diff
@@ -0,0 +1,112 @@
+    </header>
+
+    <main>
+        <section class="blue-top">
+            <div id="top" class="blue-container" >
+                <div class="text-box">
+                    <h1>일상의 모든 물건을 <br>거래해 보세요</h1>
+                    <div><a href="items.html">구경하러 가기</a></div>
+                </div>
+                <img id="img-home-top" src="img/Img_home_top.svg" alt="홈 탑 이미지">
+            </div>
+        </section>
+
+        <section class="popular">
+            <div class="white-container">
+                <div class="content-box">
+                    <img src="/img/Img_home_01.svg" alt="home img 1">
+                    <div class="texts">
+                        <p class="category">Hot item</p>
+                        <h1>인기 상품을<br>확인해 보세요</h1>
+                        <p class="description">
+                            가장 HOT한 중고거래 물품을<br>
+                            판다 마켓에서 확인해보세요
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="search">
+            <div class="white-container">
+                <div class="content-box">
+                    <div class="texts">
+                        <p class="category">Search</p>
+                        <h1>구매를 원하는<br>상품을 검색하세요</h1>
+                        <p class="description">
+                            구매하고 싶은 물품은 검색해서<br>
+                            쉽게 찾아보세요
+                        </p>
+                    </div>
+                    <img src="/img/Img_home_02.svg" alt="home img 2">
+                </div>
+            </div>
+        </section>
+
+        <section class="register">
+            <div class="white-container">
+                <div class="content-box">
+                    <img src="/img/Img_home_03.svg" alt="home img 3">
+                    <div class="texts second">
+                        <p class="category">Register</p>
+                        <h1>판매를 원하는<br>상품을 등록하세요</h1>
+                        <p class="description">
+                            어떤 물건이든 판매하고 싶은 상품을<br>
+                            쉽게 등록하세요
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="blue-bottom">
+            <div id="bottom" class="blue-container" >
+                <h1>믿을 수 있는<br>판다마켓 중고거래</h1>
+                <img src="img/Img_home_bottom.svg" alt="홈 바텀 이미지">
+            </div>
+        </section>
+    </main>
+
+    <footer>
+        <div class="footer-conatiner">
+            <span>@codeit·2024</span>
+            <div class="footer-link">
+                <a href="privacy.html">Privacy Policy</a>
+                <a href="faq.html">FAQ</a>
+            </div>
+            <div class="footer-button">
+                <a href="https://facebook.com" target="_blank ">
+                    <img id="facebook" src="/img/facebook_w.svg" alt="facebook icon">
+                </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="footer-button">
              <li>
                  <a href="https://facebook.com" target="_blank ">
                      <img id="facebook" src="/img/facebook_w.svg" alt="facebook icon">
                  </a>
                </li>
```